src/item.cpp

```diff
@@ -3085,7 +3085,10 @@ void item::on_wield( player &p, int mv )
         handle_pickup_ownership( p );
     }
     p.add_msg_if_player( m_neutral, msg, tname() );
-    p.martialart_use_message();
+
+    if( p.style_selected.obj().name != "No style" ) {
```

Comment:
This comparison is brittle. `name` is the translated name and will be different when playing with another language. I suggest you check the id of the style:
```suggestion
    if( p.style_selected != mastyle_id::NULL_ID() ) {
```
Same above.